README.md

```diff
@@ -5,37 +5,39 @@
 [Core ML Tools](https://coremltools.readme.io/docs)
 =======================
 
-Core ML is an Apple framework to integrate machine learning models into your
-app. Core ML provides a unified representation for all models. Your app uses
-Core ML APIs and user data to make predictions, and to fine-tune models, all on
-the user’s device. Core ML optimizes on-device performance by leveraging the
-CPU, GPU, and Neural Engine while minimizing its memory footprint and power
-consumption. Running a model strictly on the user’s device removes any need for
-a network connection, which helps keep the user’s data private and your app
-responsive.
-
-[Core ML tools](https://coremltools.readme.io/docs#what-is-coremltools) contains all supporting tools for [Core ML model
-conversion](https://coremltools.readme.io/docs), editing and validation. This includes deep learning frameworks like
-TensorFlow, PyTorch, Keras, Caffe as well as classical machine learning
-frameworks like LIBSVM, scikit-learn, and XGBoost.
+Use *coremltools* to convert machine learning models from third-party libraries to the Core ML format. The Python package contains the supporting tools for converting models from training libraries such as the following:
+
+* [TensorFlow 1](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf) and [TensorFlow 2](https://www.tensorflow.org/api_docs)
+* [PyTorch](https://pytorch.org/)
+* [TensorFlow's Keras APIs](https://keras.io/)
+* [Multi-backend Keras](https://github.com/keras-team/keras) 
+* [Open Neural Network eXchange](http://onnx.ai/) (ONNX)
+* Classical frameworks such as [Caffe](https://caffe.berkeleyvision.org/model_zoo.html), [LibSVM](https://www.csie.ntu.edu.tw/~cjlin/libsvm/), [scikit-learn](https://scikit-learn.org/stable/), and [XGBoost](https://xgboost.readthedocs.io/en/latest/)
```

Comment:
Caffe is not a classical ML framework. So that should probably be its' own built point. 

I think "Non-neural network" is more descriptive than "Classical".

I'd probably use sub bullet points here and order them by popularity. 

So I'd change this line to the following

-----
* [Caffe](https://caffe.berkeleyvision.org/model_zoo.html)
* Non-neural network frameworks:
    * [scikit-learn](https://scikit-learn.org/stable/)
    * [XGBoost](https://xgboost.readthedocs.io/en/latest/)
    * [LibSVM](https://www.csie.ntu.edu.tw/~cjlin/libsvm/)
-----
The code for that looks like
```
* [Caffe](https://caffe.berkeleyvision.org/model_zoo.html)
* Non-neural network frameworks:
    * [scikit-learn](https://scikit-learn.org/stable/)
    * [XGBoost](https://xgboost.readthedocs.io/en/latest/)
    * [LibSVM](https://www.csie.ntu.edu.tw/~cjlin/libsvm/)
```

Reply:
OK thanks! Done.